packages/flutter_blue/tizen/external/.clang-format

```diff
@@ -0,0 +1,2 @@
+DisableFormat: true
+SortIncludes: false
```

Comment:
Should you specify the `SortIncludes` value? (Isn't `DisableFormat` enough?)

Reply:
For some reason after removing `SortIncludes` the analysis fails.